resources/base/Receipt.activity.bxb

```diff
@@ -1,7 +1,13 @@
 activity-support {
   match: Receipt (this)
 
-  time (this.item.dateInterval.start)
+  time {
+    if (exists((this.item.dateInterval.start))) {
+      expression (this.item.dateInterval.start)
+    } else {//by definition dateInterval should have a start & end, so this path is not expected
```

Comment:
i dont think this kind of suggestion is good to have in a sample capsule

Reply:
should we just leave it with a warning then?

Reply:
what if we only have  `expression(now())` and remove if-else?

Reply:
the anchor should really be on `this.item.dateInterval.start`

Reply:
alright. then lets go with your suggestion

Reply:
What does it mean that by definition it has a start and end? there are open intervals. you can have just a start or an end.

Reply:
TIL: "starting next week" type of pattern can lead to only having one of either start or an end even for dateInterval concept.

Reply:
removed comment